ci/release.js

```diff
@@ -88,20 +90,35 @@ function release(pomVersion) {
     ci.sh('mvn -B -s ci/settings.xml clean release:prepare release:perform -DreleaseVersion=' + newVersion + ' -DdevelopmentVersion=' + nextVersion);
     
     ci.stage('RELEASE DONE');
+    return newVersion;
 }
 
-function commit() {
-    // The maven release:perform goal does not commit the Javascript, Swagger, and Nginx generated files
-    // so we do this here after the release
+/**
+ * Updates the Swagger UI index.html file with the newly release Swagger specification
+ * 
+ * @param {*} releasedVersion The newly released version.
+ */
+function updateSwaggerUiIndex(releasedVersion) {
+    // The release tag is 1 commit before the last maven release:prepare commit
+    let releaseCommit = cp.execSync(`git rev-parse HEAD~1`).toString().trim();
 
-    ci.stage('COMMIT - Adding/staging/deleting all changes to the generated files');
+    let index = fs.readFileSync(__dirname + '/../docs/index.html', 'utf-8');
 
-    ci.sh('git add -A src/main/resources/javascript/*');
-    ci.sh('git add -A src/main/resources/swagger/*');
-    ci.sh('git add -A src/main/resources/nginx/*');
-    ci.sh('git add -A docs/swagger.json');
-    ci.sh('git commit -m "@releng : automatically commit generated files after maven release:perform"');
+    // Parses the urls array, adds the new Swagger link, and replaces the current file 
+    let urls = eval(index.match(/urls: (\[[\s\S]*?\])/)[1]);
```

Comment:
This looks quite fragile to me. What about listing all the git tags with `git tag -l api-model-*` and add all the links to it each time (e.g. https://raw.githubusercontent.com/adobe/commerce-cif-api/api-model-1.1.0/docs/swagger.json)? 
In this way, you can easily execute the job again if something went wrong without risking duplicates.